docs/alerting-recording.md

```diff
@@ -0,0 +1,59 @@
+# Alerting & Recording Rules
+
+### Alerting Rules
+
+The alerting rules are used to trigger alerts based on the provided alerting rules an alert should be fired to an external service like slack. In promscale the alerting rules are evaluated in prometheus. i.e define alerting rules in prometheus config
```

Comment:
1. This sentence does not sound good.
2. We do not define alerting rules in the prometheus configuration file. We mention the alerting-rules file name as a list where these files contains the actual rules. This statement needs an update in this direction.

Reply:
```suggestion
Alerting rules are used to trigger alerts based on the violation of any condition(s). These alerts are fired to external services like slack, mails, etc by the alert manager (link). Alerting rules are written in a YAML file and paths of these files are mentioned in the Prometheus configuration respectively. It is important to note that the evaluation of these conditional rules are done on the Prometheus side.
```